src/kboot.c

```diff
@@ -1596,6 +1661,72 @@ static int dt_set_display(void)
     return dt_vram_reserved_region("dcp", "disp0");
 }
 
+static int dt_set_sio_fwdata(void)
+{
+    const char *path = "/soc/sio";
```

Comment:
Shouldn't this be an alias? The paths aren't stable (e.g. t600x have the dies)

Reply:
You mean shouldn't we be looking at some aliased path other than `/soc/sio`? AFAIK this will work if `/soc/sio` itself is an alias.

Reply:
Aliases are bare labels (like just "sio"), not paths. They point at paths :)

I think just `fdt_path_offset(dt, "sio");` should work, as long as you then add a corresponding `sio` alias (look at `dcp` and `gpu` for examples)